airbyte-integrations/connectors/source-stripe/Dockerfile

```diff
@@ -1,16 +1,15 @@
-FROM airbyte/integration-base-python:0.1.5
+FROM python:3.7-slim
 
 # Bash is installed for more convenient debugging.
 RUN apt-get update && apt-get install -y bash && rm -rf /var/lib/apt/lists/*
 
-ENV CODE_PATH="source_stripe"
-ENV AIRBYTE_IMPL_MODULE="source_stripe"
-ENV AIRBYTE_IMPL_PATH="SourceStripe"
-
 WORKDIR /airbyte/integration_code
-COPY $CODE_PATH ./$CODE_PATH
+COPY source_stripe ./source_stripe
+COPY main.py ./
 COPY setup.py ./
 RUN pip install .
 
-LABEL io.airbyte.version=0.1.4
+ENTRYPOINT ["python", "/airbyte/integration_code/main.py"]
```

Comment:
This is sweet!

Guessing we'll clean up the old entry point PR once we've migrated everything to the new CDK?

Reply:
the old base images? yup